src/managers/toolkitGlobal.ts

```diff
@@ -0,0 +1,20 @@
+import { ItemTreeExtraColumnsGlobal } from "./itemTree";
+
+export default class ToolkitGlobal {
+    readonly itemTree: ItemTreeExtraColumnsGlobal;
+
+    private constructor() {
+        this.itemTree = {
+            initialized: false,
+            columns: [],
+            fieldHooks: {},
+            renderCellHooks: {},
+        };
+    }
+
+    static getInstance(zotero: _ZoteroTypes.Zotero): ToolkitGlobal {
```

Comment:
Discussion: getInstance here, or getGlobal in basicTool?

Reply:
you can add a wrap for this in `getGlobal`

Reply:
In addition, `getInstance` here is necessary for Singleton.